demos/general/demo_OnACID_mesoscope.py

```diff
@@ -3,7 +3,7 @@
 
 """
 Complete pipeline for online processing using CaImAn Online (OnACID).
-The demo demonstates the analysis of a sequence of files using the CaImAn online
+The demo demonstrates the analysis of a sequence of files using the CaImAn online
```

Comment:
I'll circle back on this afterwards and fix the wording ("demo demonstrates" has the feel of "ATM machine") unless you want to adjust it